### DISCUSSION TITLE
Installation Scripting or Setting FIle

### DISCUSSION THREAD

**Discussion by DRPRFS-CSBSJU** [thonny/thonny#3351] [General]
We use Thonny on an institutional level. Our IT is looking for either:
* Installer flags/arguments
* an INI/settings file
* A script we can run post-install

that can be used to do either/both of the following:

* Pre-install a list of packages with Thonny
* Change configuration settings (like turning the assistant options off)
Because we do institutional-wide installs, they currently use the default configuration in a scripted install, but we'd like to customize things a bit.

**Comment by aivarannamaa** [comment]
You may find something here:

* https://jrsoftware.org/ishelp/index.php?topic=setupcmdline
* https://github.com/thonny/thonny/wiki/DeploymentOptions
